ember-toucan-core/src/components/form/input-field.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="flex flex-col {{if @isDisabled 'text-disabled cursor-not-allowed'}}">
```

Comment:
Note: none of this is the finished styling currently, as we are waiting on user and design research. Styling will be for an upcoming iteration.